front/src/components/CTF/DiscordEventLinkSync.vue

```diff
@@ -1,14 +1,18 @@
 <template>
   <q-input
     v-model="discordEventLink"
+    dense
+    filled
     bottom-slots
-    label="Discord Event Link"
+    label="Discord event link"
     hint="Format: https://discord.gg/abcdefg?event=123456789"
-    filled
   >
```

Comment:
Since there is still space left, maybe it is nice to increase the width of the `<q-input>` to always fit a Discord event link without scrolling? This was also not the case in the old design however.